landoui/template_helpers.py

```diff
@@ -202,6 +202,9 @@ def bug_url(text):
 
 @template_helpers.app_template_filter()
 def revision_url(text, diff_id=None):
+    if not text.startswith("D"):
+        text = f"D{text}"
+
```

Comment:
Should we check that `text` can be interpreted as a valid `int`?

Reply:
Hmm good point, I think this can be improved. Perhaps not necessary to check the `int` interpretation, and more important to check that we can indeed parse this as a string.